packages/verkle/src/node/internalNode.ts

```diff
@@ -1,39 +1,49 @@
-import { equalsBytes } from '@ethereumjs/util'
+import { BIGINT_0, bytesToBigInt, equalsBytes } from '@ethereumjs/util'
 
+import { type VerkleCrypto, zeroValues } from '../types.js'
 import { POINT_IDENTITY } from '../util/crypto.js'
 
 import { BaseVerkleNode } from './baseVerkleNode.js'
 import { LeafNode } from './leafNode.js'
 import { NODE_WIDTH, VerkleNodeType } from './types.js'
 
-import type { VerkleNode, VerkleNodeOptions } from './types.js'
+import type { VerkleNodeOptions } from './types.js'
 
 export class InternalNode extends BaseVerkleNode<VerkleNodeType.Internal> {
-  // Array of references to children nodes
-  public children: Array<VerkleNode | null>
-  public copyOnWrite: Record<string, Uint8Array>
+  // Array of uncompressed commitments (i.e. 64 byte Uint8Arrays) to child nodes
+  public children: Uint8Array[]
   public type = VerkleNodeType.Internal
 
-  /* TODO: options.children is not actually used here */
   constructor(options: VerkleNodeOptions[VerkleNodeType.Internal]) {
     super(options)
-    this.children = options.children ?? new Array(NODE_WIDTH).fill(null)
-    this.copyOnWrite = options.copyOnWrite ?? {}
+    this.children = options.children ?? new Array(64).fill(new Uint8Array(64))
```

Comment:
Just reading up the conversation with Kev. I get why children are 64-bytes in length (uncompressed version), but I'm not sure why the length of the array is itself 64, shouldn't it be 256 since an internal node can have 256 children?

Reply:
That's a bug. Should be 256.